application/src/pages/Cart/CarInCart/CarInCart.tsx

```diff
@@ -0,0 +1,41 @@
+import React, { ChangeEvent } from 'react';
+import { useDispatch } from 'react-redux';
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
+import { faTrashAlt } from '@fortawesome/free-regular-svg-icons';
+import {
+  CartPropTypes,
+  addAmount,
+  deleteCar
+} from '../../../bll/reducers/cartReducer';
+import style from './CarInCart.module.css';
+
+export const CarInCart: React.FC<CartPropTypes> = ({ ...props }) => {
+  const dispatch = useDispatch();
+
+  const setNewAmount = (e: ChangeEvent<HTMLInputElement>) => {
+    dispatch(addAmount(props.id, Number(e.currentTarget.value)));
+  };
+
+  const handleDeleteCar = () => {
+    dispatch(deleteCar(props.id));
+  };
+  const total = Math.floor(props.total * 100) / 100;
```

Comment:
You have duplication of Math.floor for total calculation
let's create common util function

```
formatTotalAmount(props.total)
...
const formatTotalAmount = (amount) => {
...
}
```

Reply:
done